content/04.discussion.md

```diff
@@ -1,26 +1,31 @@
 ## Discussion
 
-Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
-We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract collaborators to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
-To our knowledge, this initiative represents the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+Through the OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 1,253 pediatric brain tumors across 59 distinct histologies.
+We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```

Comment:
```suggestion
We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as to attract interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
```

What do we think about "entice" vs "attract"?

Reply:
we can go with "entice"